Lib/test/test_sysconfig.py

```diff
@@ -394,6 +395,24 @@ def test_ldshared_value(self):
 
         self.assertIn(ldflags, ldshared)
 
+    @unittest.skipIf(not _imp.extension_suffixes(), "stub loader has no suffixes")
+    def test_soabi(self):
+        soabi = sysconfig.get_config_var('SOABI')
+        self.assertIn(soabi, _imp.extension_suffixes()[0])
+
+    def test_library(self):
+        library = sysconfig.get_config_var('LIBRARY')
+        ldlibrary = sysconfig.get_config_var('LDLIBRARY')
+        major, minor = sys.version_info[:2]
+        if sys.platform == 'win32':
+            self.assertTrue(library.startswith(f'python{major}{minor}'))
+            self.assertTrue(library.endswith('.dll'))
+            self.assertEqual(library, ldlibrary)
+        else:
+            self.assertTrue(library.startswith(f'libpython{major}.{minor}'))
+            self.assertTrue(library.endswith('.a'))
+            self.assertTrue(ldlibrary.startswith(f'libpython{major}.{minor}'))
+
```

Comment:
(nitpick)

Ideally, we should load the `SOABI` and `LIBRARY`/`LDLIBRARY` values from the C macros, but if we don't, we could at least compare the value we generate against them — by writing a C test for eg.

Let's figure out the data source first, though.